package.json

```diff
@@ -48,10 +48,10 @@
     "@commitlint/cli": "^13.1.0",
     "@opentelemetry/sdk-node": "^0.24.0",
     "@commitlint/config-conventional": "^13.1.0",
-    "@nestjs/common": "^8.0.5",
-    "@nestjs/core": "^8.0.5",
-    "@nestjs/platform-express": "^8.0.5",
-    "@nestjs/testing": "^8.0.5",
+    "@nestjs/common": "^7.0.0 || ^8.0.0",
```

Comment:
I believe you only need the peer Dependencies to be updated. Those are for local development, right?

Reply:
Yes, however I kept it consistent with peer-dependency to ensure that there are no surprises during development process. 

On second thought, I wonder how during development process, we detect any breaking API changes between NestJS V7 and V8. 
I will revert them.